src/backends/gstreamer/audio_stream.rs

```diff
@@ -1,31 +1,54 @@
 use AudioStream;
 use gst::prelude::*;
-use super::gst_player;
-use super::glib;
+use super::gst;
+
+use std::process;
 
 // XXX Define own error type.
 
 pub struct GStreamerAudioStream {
-    player: gst_player::Player,
+    pipeline: gst::Element,
 }
 
 impl GStreamerAudioStream {
     pub fn new() -> Result<Self, ()> {
-        let player = gst_player::Player::new(None, None);
-        player
-            .set_property("uri", &glib::Value::from("webaudiosrc://foo"))
-            .expect("Cant't set URI property");
-        Ok(Self { player: player })
+        if let Some(category) = gst::DebugCategory::get("gst-player") {
```

Comment:
This is useless now, sorry I missed it in the previous patch :) Anyway just a nit